courses/fundamentals_of_ada/labs/120_limited_types.lab.rst

```diff
@@ -19,164 +19,28 @@ Limited Types Lab
 Limited Types Lab Solution - Employee Data (Spec)
```

Comment:
Why not continue with the (XXX) -> "- XXX" trend?

Reply:
This one is the format I want, but others aren't, so I'll make them consistent.
My goal:
  for subprogram solutions. slide title should be "<section> Lab Solution - <description>"
  for package solutions, slide title should be "<section> Lab Solution - <package name> [(Spec|Body)]"